transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -168,8 +167,8 @@ protected void run() {
                         }
                     }
                 } finally {
-                    if (nextWakeupNanos.get() == AWAKE || nextWakeupNanos.getAndSet(AWAKE) == AWAKE) {
-                        pendingWakeup = true;
+                    if (nextWakeupNanos.get() == AWAKE) {
```

Comment:
sorry... missed this one. This if makes no sense. Remove it and just call set every time